lighthouse-core/test/computed/bundle-analysis-test.js

```diff
@@ -0,0 +1,347 @@
+          "webpack:///./src/codecs/resize/options.tsx": 3970,
+          "webpack:///./src/codecs/resize/processor-meta.ts": 225,
+          "webpack:///./src/codecs/resize/processor-sync.ts": 462,
+          "webpack:///./src/codecs/resize/util.ts": 134,
+          "webpack:///./src/codecs/rotate/processor-meta.ts": 18,
+          "webpack:///./src/codecs/tiny.webp": 89,
+          "webpack:///./src/codecs/webp/encoder-meta.ts": 660,
+          "webpack:///./src/codecs/webp/options.tsx": 5114,
+          "webpack:///./src/components/Options/index.tsx": 2176,
+          "webpack:///./src/components/Options/style.scss": 410,
+          "webpack:///./src/components/Output/custom-els/PinchZoom/index.ts": 3653,
+          "webpack:///./src/components/Output/custom-els/TwoUp/index.ts": 2088,
+          "webpack:///./src/components/Output/custom-els/TwoUp/styles.css": 75,
+          "webpack:///./src/components/Output/index.tsx": 5199,
+          "webpack:///./src/components/Output/style.scss": 447,
+          "webpack:///./src/components/checkbox/index.tsx": 247,
+          "webpack:///./src/components/checkbox/style.scss": 106,
+          "webpack:///./src/components/compress/custom-els/MultiPanel/index.ts": 3461,
+          "webpack:///./src/components/compress/custom-els/MultiPanel/styles.css": 105,
+          "webpack:///./src/components/compress/index.tsx": 8782,
+          "webpack:///./src/components/compress/result-cache.ts": 611,
+          "webpack:///./src/components/compress/style.scss": 132,
+          "webpack:///./src/components/expander/index.tsx": 901,
+          "webpack:///./src/components/expander/style.scss": 66,
+          "webpack:///./src/components/range/index.tsx": 566,
+          "webpack:///./src/components/range/style.scss": 200,
+          "webpack:///./src/components/results/FileSize.tsx": 445,
+          "webpack:///./src/components/results/index.tsx": 1538,
+          "webpack:///./src/components/results/style.scss": 780,
+          "webpack:///./src/components/select/index.tsx": 291,
+          "webpack:///./src/components/select/style.scss": 103,
+          "webpack:///./src/custom-els/RangeInput/index.ts": 2138,
+          "webpack:///./src/custom-els/RangeInput/styles.css": 180,
+          "webpack:///./src/lib/clean-modify.ts": 331,
+          "webpack:///./src/lib/icons.tsx": 2531,
+          "webpack:///./src/lib/util.ts": 4043,
+        },
+        "totalBytes": 83748,
+        "unmappedBytes": 10061,
+      }
+    `);
+
+    // Test the mapping.
+    const entry = result.map.findEntry(0, 80476);
+    expect(entry).toMatchInlineSnapshot(`
+      SourceMapEntry {
+        "columnNumber": 80469,
+        "lastColumnNumber": 80482,
+        "lineNumber": 0,
+        "name": "workerResize",
+        "sourceColumnNumber": 31,
+        "sourceLineNumber": 119,
+        "sourceURL": "webpack:///./src/components/compress/index.tsx",
+      }
+    `);
+  });
+
+  it('handles faulty maps', async () => {
```

Comment:
It's tricky to create "meaningful" bad source maps, so I just mutate a bunch of things.